managed/services/management/rds.go

```diff
@@ -120,7 +120,7 @@ func listRegions(partitions []string) []string {
 				continue
 			}
 
-			for r := range partition.Services()[endpoints.RdsServiceID].Regions() {
+			for r := range partition.Services()[rds.EndpointsID].Regions() {
```

Comment:
Note: those two constants are equivalent, so this replacement should't change the behavior.